clients/admin-ui/src/features/common/Header.tsx

```diff
@@ -59,20 +66,34 @@ const Header: React.FC = () => {
                 size="sm"
                 variant="ghost"
                 data-testid="header-menu-button"
+                onDoubleClick={() => featuresPanelDisclosure.onOpen()}
```

Comment:
Pro-tip: I'm leaving this here without the flag check so that those who know the trick can access the panel in prod builds even when the menu entry isn't shown.